test/tools/cmap_spec_runner.ts

```diff
@@ -135,31 +137,35 @@ class Thread {
  * Implements the spec test match function, see:
  * [CMAP Spec Test README](https://github.com/mongodb/specifications/tree/master/source/connection-monitoring-and-pooling/tests#spec-test-match-function)
  */
-const compareInputToSpec = (input, expected) => {
+const compareInputToSpec = (input, expected, message) => {
   // the spec uses 42 and "42" as special keywords to express that the value does not matter
   // however, "42" does not appear in the spec tests, so only the numeric value is checked here
   if (expected === 42) {
-    expect(input).to.be.ok; // not null or undefined
+    expect(input, message).to.be.ok; // not null or undefined
```

Comment:
```suggestion
expect(input, message).to.not.be.null;
expect(input, message).to.not.be.undefined;
```

Should we narrow this to gaurd against the other falsey values in JS that we **woudn't** want accepted here?

Reply:
We should have caught this earlier, but better late than never :) - thought for some reason that it was just null/undefined. We do *not* want to throw on falsy values here unless they're explicitly null or undefined (per the [spec match function description](https://github.com/mongodb/specifications/tree/master/source/connection-monitoring-and-pooling/tests#spec-test-match-function)). I'll update to your suggestion, but not via github because I think the spacing will be off.